Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -1,37 +1,472 @@
-﻿namespace BootCamp.Chapter
+﻿using System.Globalization;
+using System.Text;
+using System;
+using System.Diagnostics;
+
+namespace BootCamp.Chapter
 {
     public static class BalanceStats
     {
+        public static string defaultCurrencySymbol = "¤";
+        public static int defaultCurrencyLocation = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4971928)